### ISSUE TITLE
Missing dependencies break npx cra common use

### ISSUE THREAD

**Issue by VictorVelarde** [CartoDB/carto-react-template#160]
I have detected that both skeleton and sample-app templates are currently broken, using node v14 LTS

Both of them failed when being used with the typical npx create-react-app flow, throwing an error like this when executing `yarn start`:

```
Failed to compile.

./src/index.js
Error: [BABEL] /Users/victor/CODE/carto-for-react-training/setup/my-app/src/index.js: Cannot find module '@babel/helper-builder-react-jsx'
Require stack:
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/babel-preset-react-app/node_modules/@babel/plugin-transform-react-jsx/lib/transform-classic.js
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/babel-preset-react-app/node_modules/@babel/plugin-transform-react-jsx/lib/index.js
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/babel-preset-react-app/node_modules/@babel/preset-react/lib/index.js
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/babel-preset-react-app/create.js
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/babel-preset-react-app/index.js
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/react-scripts/node_modules/@babel/core/lib/config/files/plugins.js
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/react-scripts/node_modules/@babel/core/lib/config/files/index.js
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/react-scripts/node_modules/@babel/core/lib/index.js
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/react-scripts/node_modules/babel-loader/lib/index.js
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/loader-runner/lib/loadLoader.js
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/loader-runner/lib/LoaderRunner.js
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/webpack/lib/NormalModule.js
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/webpack/lib/NormalModuleFactory.js
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/webpack/lib/Compiler.js
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/webpack/lib/webpack.js
- /Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/react-scripts/scripts/start.js (While processing: "/Users/victor/CODE/carto-for-react-training/setup/my-app/node_modules/babel-preset-react-app/index.js")
```

**Comment by VictorVelarde** [comment]
I have seen that this error appears for 2 "missing" dependencies: **@babel/plugin-transform-react-jsx** and **@babel/helper-builder-react-jsx**

Adding both of them and running `yarn start` again seems to fix the error:
```
yarn add @babel/plugin-transform-react-jsx
yarn add @babel/helper-builder-react-jsx
yarn start
```

**Comment by VictorVelarde** [comment]
My suspect is that a dependency has changed before our latest release (currently 1.0.0-beta9). It might be due to the latest `react-app-rewired` update, in mid-December 2020, (as we express the dependency with "react-app-rewired": "^2.1.6")

**Comment by VictorVelarde** [comment]
Solved with v1.0.0-beta10
